test/built-ins/Atomics/wait/true-for-timeout.js

```diff
@@ -38,8 +38,11 @@ var int32Array = new Int32Array(new SharedArrayBuffer(Int32Array.BYTES_PER_ELEME
 
 $262.agent.broadcast(int32Array.buffer);
 
-$262.agent.sleep(2);
+$262.agent.sleep(150);
```

Comment:
?

Reply:
Will revert -- the sleep in the while loop covers for the minimum required wait time